### ISSUE TITLE
Incompatible with Yosemite?

### ISSUE THREAD

**Issue by fullofcaffeine** [ncannasse/castle#18]
I just downloaded CastleDB from its website, unzipped and tried running it on my MacBook Pro 15" retina running Yosemite, and I got the following generic error message:

https://www.dropbox.com/s/adodeqsmdv56if4/Screenshot%202015-03-23%2012.19.30.png?dl=0

Any hints?

**Comment by ncannasse** [comment]
OSX give some useful error messages...
Could you try to check rights (chmod +x ?)

**Comment by fullofcaffeine** [comment]
Hi Nicolas,

Looks like this was one of the issues. I made the node-webkit binary (in ```castleDB.app/Contents/MacOS```) executable, but now the app just silently crashes on itself.

Here's the ouput when running the node-webkit bin from the CLI:

```
./node-webkit                                                                                          [2.1.5]
2015-03-30 11:27:11.888 node-webkit[16361:9163831] Internals of CFAllocator not known; out-of-memory failures via CFAllocator will not result in termination. http://crbug.com/45650
libc++abi.dylib: terminating with uncaught exception of type std::logic_error: basic_string::_S_construct NULL not valid
[1]    16361 abort      ./node-webkit
```

**Comment by ncannasse** [comment]
Try to update to latest node webkit here : https://github.com/nwjs/nw.js, or report them the issue

**Comment by ncannasse** [comment]
I have released a 1.5, if you want to try again.
